LondonBritishLibrary/orient/BLorient808.xml

```diff
@@ -30,22 +30,62 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 371</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
-                        <summary/>
-                        
-                        
-                        
-                    </msContents>
-                    
-                    
-                    
-                    
-                    
-                    
+<msItem xml:id="ms_i1">
+<locus from="1r"/>
+<title type="complete" ref="LIT1395Fethan"/>
+<textLang mainLang="gez"/>
+
+<msItem xml:id="ms_i1.1">
+<locus from="1r"/>
+<title type="complete" ref="LIT1395Fethan#Introduction"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.2">
+<locus from="4v"/>
+<title type="complete" ref="LIT1395Fethan#index"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.3">
+<locus from="5r"/>
+<title type="complete" ref="LIT1395Fethan#Part1"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.4">
+<locus from="58v"/>
+<title type="complete" ref="LIT1395Fethan#Part2"/>
+<textLang mainLang="gez"/>
+</msItem>
+
+<msItem xml:id="ms_i1.5">
+<locus from="128r"/>
+<title xml:lang="gez" type="complete" ref="LIT1395Fethan#Appendix">አንቀጸ፡ ተዋርሶ፡</title>
+<textLang mainLang="gez"/>
+</msItem>
+
+<colophon xml:id="coloph1">
+መልአ፡ ጽሒፈ፡ ዝንቱ፡ መጽሐፈ፡ ቀኖና፡ ዘፍትሐ፡ ኵነኔ፡ ሕግ፡ መሲሓዊት፡ ወእምዓመተ፡ ሥጋዌ፡ <date calendar="ethiopian" when="1762">፲፻ወ፯፻፷ወ፪፡</date> ወበ፲፻፡ ወ
+በ፫፻፹ወ፫ዓመተ፡ ሰማዕታት። ወእምተንባላት፡ በ<date calendar="islamic" when="1020">፲፻ወ፳ዓመት</date>። ወተጽሐፈ፡ በዘመነ፡ ማርሮስ፡ ወንጌላዊ፡ በ፩አበቅቴ፡
+ፀሐይ፡ ወበ፩አበቅቴ፡ ወርኅ። ወበ፪ዓመተ፡ መንግሥቱ፡ ለንጉሠ፡ ሃይማኖት፡<gap reason="omitted"/>ወለእግዚእነ፡ አኰቴት፡ እስከ፡ ለዓለም።
+<note>The name of the owner <persName role="owner" ref="PRS13228WaldaMaryam">Walda Māryām</persName> is given in arithmetical figures i. e. ፷፪፻፬፯፺፬.</note>
+</colophon>
```

Comment:
That is interesting! Is the name mentioned in the colophon?

Reply:
It's hard to tell, see p. 282b.